### DISCUSSION TITLE
How to connect your own user system with the user system of nocodb

### DISCUSSION THREAD

**Discussion by zhaohaodong** [nocodb/nocodb#4202] [Q&A]
How to connect your own user system with the user system of nocodb

**Comment by dstala** [comment]
@zhaohaodong 
Can you elaborate on your requirement?

**Reply by zhaohaodong** [reply]
Thank you for your reply. My requirements are as follows: I see that your user information and permissions need to be created in nocodb, but we already have users in our own system. We just want to use the functions related to tables, and do not need users to register themselves. Our system controls user permissions. For example, nocodb is a plug-in integrated into our system
![image](https://user-images.githubusercontent.com/32354104/198965410-ff3af857-6ef5-4ae0-9789-672bd74c9ea0.png)
